package.json

```diff
@@ -32,16 +32,15 @@
   "homepage": "https://github.com/libp2p/js-libp2p-mdns",
   "devDependencies": {
     "aegir": "^18.0.2",
-    "async": "^2.6.1",
     "chai": "^4.2.0",
     "dirty-chai": "^2.0.1"
   },
   "dependencies": {
     "libp2p-tcp": "~0.13.0",
     "multiaddr": "^6.0.2",
     "multicast-dns": "^7.2.0",
-    "peer-id": "~0.12.0",
-    "peer-info": "~0.15.0"
+    "peer-id": "libp2p/js-peer-id#feat/async-await",
+    "peer-info": "libp2p/js-peer-info#feat/async-await"
```

Comment:
I see, this PR is blocked on the release of peer-id and peer-info

Reply:
Actually nvm. It is done

- https://github.com/libp2p/js-peer-id/pull/87 on 0.13
- https://github.com/libp2p/js-peer-info/pull/67 on 0.16